src/style.css

```diff
@@ -1 +1,93 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-link-active: #00acdc;
+  --color-link: #000;
+  --color-background-container: #fff;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  /* line-height: 14px; */
+}
+
+ul {
+  list-style: none;
+}
+
+.container {
+  background-color: var(--beckground-container);
```

Comment:
Typo in variable name, change '--beckground-container' to '--color-background-container'